staking/test/contracts/StakingTest.sol

```diff
@@ -9,23 +9,34 @@ contract StakingTest {
     event createLockResult(uint result);
     event balanceOfResult(uint result);
     event totalBalanceResult(uint result);
+    event reStakeResult(uint result);
 
-    function _createLock(address staking, address account, uint amount, uint period, uint cliff) external {
+    function _stake(address staking, address locker, address delegate, uint amount, uint slope, uint cliff) external {
```

Comment:
Do you need this contract? Why Stake can't be used without this StakingTest?

Reply:
I use event emit to return resultsof contract Staking methods  in *test.js. I cant found another way. So I need StakingTest.